tests/scripts/thread-cert/pktverify/layer_fields.py

```diff
@@ -42,7 +42,7 @@
 
 def _auto(v: Union[LayerFieldsContainer, LayerField]):
     """parse the layer field automatically according to its format"""
-    assert not isinstance(v, LayerFieldsContainer) or len(v.fields) == 1, v.fields
+    assert not isinstance(v, LayerFieldsContainer) or len(v.fields) == 1 or v.get_default_value() is not None, v.fields
```

Comment:
What's the purpose of this change?

Reply:
The assertion failed without this condition. I met a case that v doesn't have field fields.

Reply:
Which field has this problem?

Reply:
`fields`

Reply:
Here's what I got without the added condition: 
```
Traceback (most recent call last):
  File "/usr/local/google/home/handaw/src/openthread/tests/scripts/thread-cert/pktverify/layer_fields.py", line 694, in get_layer_field
    v = _LAYER_FIELDS[field_uri](v)
  File "/usr/local/google/home/handaw/src/openthread/tests/scripts/thread-cert/pktverify/layer_fields.py", line 45, in _auto
    assert not isinstance(v, LayerFieldsContainer) or len(v.fields) == 1, v.fields
AssertionError: [<LayerField thread_nwd.tlv.server.16: 0x00001c00>, <LayerField thread_nwd.tlv.server.16: 0x00001c00>]

The above exception was the direct cause of the following exception:

```

Reply:
Here's what I got without the added condition: 
```
Traceback (most recent call last):
assert not isinstance(v, LayerFieldsContainer) or len(v.fields) == 1, v.fields
AssertionError: [<LayerField thread_nwd.tlv.server.16: 0x00001c00>, <LayerField thread_nwd.tlv.server.16: 0x00001c00>]

The above exception was the direct cause of the following exception:

```

Reply:
Seems there is multiple `thread_nwd.tlv.server.16` field in this layer. 

Maybe we should change the layer field from `_auto` to `_list(_auto)` ? @superwhd